content/en/notebooks/_index.md

```diff
@@ -30,21 +30,16 @@ You can create a notebook in two places:
 - From the left-hand navigation bar, click **Dashboards > New Notebook**.
 - On the upper-right-hand corner of the [Notebooks List page][1], click **New Notebook**.
 
-{{< img src="/notebooks/notebooks_new/new_legacy_notebook_dropdown.png" alt="Create legacy notebook through the New Notebook dropdown" style="width:100%;" >}}
-
-To create a legacy notebook, click the **New Noteboook** dropdown and select **New Legacy Notebook**. 
-
 ### Notebook templates
 
 In the [Template Gallery][2], see ready-to-use templates which you can create new notebooks from. Templates include an Incident Response postmortem, an Incident Report, and SLO Specification. You can also create a new custom template to build reusable notebook structures.
 
 ## Editing a notebook
 
-With Notebooks, you can start typing using markdown shortcuts such as `#` for headers or <code>```</code> for code blocks. Notebooks comprise cells that can contain text or graphs. A new cell is created every time you press <kbd>Enter</kbd>. Notebooks automatically save your text content as you type. For graph cells, save all edits in the graph editor for the widget to be saved in the notebook.
+With Notebooks, you can start typing using markdown shortcuts such as `#` for headers or <code>```</code> for code blocks. Notebooks automatically save your text content as you type. For graphs, save all edits in the graph editor for the widget to be saved in the notebook.
```

Comment:
```suggestion
With Notebooks, you can start typing using markdown shortcuts, such as `#` for headers or <code>```</code> for code blocks. Notebooks automatically save your text content as you type. For graphs, save all edits in the graph editor for the widget to be saved in the notebook.
```
I noticed a missing comma in the existing text.